src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/dto/BeregningsresultatBoutgifterDto.kt

```diff
@@ -86,7 +86,12 @@ fun BeregningsresultatForLøpendeMåned.finnUtgifterMedAndelTilUtbetaling(
         .flatten()
         .sorted()
         .map { utgift ->
-            val skalUtbetales = minOf(utgift.utgift, grunnlag.makssats - totalSum)
+            val skalUtbetales =
+                if (grunnlag.skalFåDekketFaktiskeUtgifter()) {
+                    utgift.utgift
+                } else {
+                    minOf(utgift.utgift, grunnlag.makssats - totalSum)
+                }
```

Comment:
Denne logikken blir duplisert i `Beregningsgrunnlag`. Skulle den vært trukket ut til noe felles-util, eller er det faktisk to ulike ting? 🤔

Reply:
Hva mener du med at den blir duplisert?